content/en/blog/2023-02-01-january-update/index.md

```diff
@@ -0,0 +1,384 @@
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
+It's important to keep you up to date with new features and developments
+in Flux and provide simple ways to see our work in action and chat with
+our engineers.
+
+### Recent Events (ICYMI) 📺
+
+We feel blessed to have such a big community of users, contributors and
+integrators and so many are happy to talk about their experiences. In
+December and January here are a couple of talks we would like to highlight.
+
+{{< youtube JHmQlSvL0II >}}
+
+> HashiCorp User Group Luxembourg: GitOps your Terraform Configurations
+>
+> Flux Terraform Controller is a controller for Flux to reconcile Terraform
+> configurations in the GitOps way with the power of Flux and Terraform,
+> Terraform Controller allows you to GitOps-ify your infrastructure, and
+> your application resources, in the Kubernetes and Terraform universe.
+>
+> Flux Terraform Controller ensures what you’ve defined in your Terraform
+> configurations is what’s always running and available. Flux continuously
+> looks for changes and reconciles with the desired state. Take advantage
+> of all the benefits of GitOps: streamlined and secure deployments, quicker
+> time to market, and more time to concentrate on app development!
+
+{{< youtube uRiCRTSkPOQ >}}
+
+> Flux’s Security & Scalability with OCI & Helm (Part 2) with Kingdon Barrett
+>
+> With Flux, you can distribute and reconcile Kubernetes configuration packaged
+> as OCI artifacts. Instead of connecting Flux to a Git repository where the
+> application desired state is defined, you can connect Flux to a container
+> registry where you’ll push the application deploy manifests, right next to
+> the application container images.
+>
+> During this session Kingdon Barrett, OSS Engineer at Weaveworks & Flux
+> Maintainer, shows you how to quickly create scalable and Cosign-verified GitOps
+> configurations with Flux using the same process with two demo environments: one
+> will be a Kustomize Environment and the other a Helm-based environment.
+
+{{< youtube Bmh7kKYLIhY >}}
+
+> Flux Security & Scalability using VS Code GitOps Extension
+>
+> Recently Flux has released two new features (OCI and Cosign) for scalable and
+> secure GitOps. Juozas Gaigalas, a Developer Experience Engineer at Weaveworks,
+> will demonstrate how developers and platform engineers can quickly create
+> scalable and Cosign-verified GitOps configurations using VS Code GitOps Tools
+> extension. New and experienced Flux users can learn about Flux’s OCI and Cosign
+> support through this demo. Join us!
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+February - tune in to learn more about Flux and GitOps best practices,
```

Comment:
@staceypotter @vanessaabankwah Anything we should bring up here for Dec+Jan+Feb?

Reply:
https://www.meetup.com/gitops-community/events/289768516 maybe?